metrics/coverage_high_water_mark

```diff
@@ -1 +1 @@
-93.6700
+93.1600
```

Comment:
This is the downside.  I'll make up for some of that in #140, for sure, but won't have time to get us back to this number.

Reply:
I do think that the complexity-reduction here is an overall benefit.